stablehlo/reference/Ops.cpp

```diff
@@ -94,6 +94,22 @@ Tensor evalClampOp(const Tensor &min, const Tensor &operand, const Tensor &max,
   return result;
 }
 
+Tensor evalConcatenateOp(ArrayRef<Tensor> inputs, int64_t dimension,
+                         TensorType resultType) {
+  Tensor result(resultType);
+  int64_t offsetDimSize = 0;
+  for (size_t inputsDim = 0; inputsDim < inputs.size(); ++inputsDim) {
```

Comment:
I don't think `inputsDim` is a great name because this seems to be an index in the inputs array, rather than anything related to dimensions. How about we just say `i`?